src/components/SeedBank/StorageLocations.tsx

```diff
@@ -36,30 +52,122 @@ export default function SubLocations({ seedBankId }: SubLocationsProps): JSX.Ele
       }
     };
 
-    if (isEnabled('Storage locations')) {
-      fetchStorageLocations();
+    if (featureEnabled) {
+      if (seedBankId) {
+        fetchStorageLocations();
+      } else {
+        setStorageLocations(DEFAULT_STORAGE_LOCATIONS().map((name, index) => storageLocationWith(name, index)));
+      }
+    }
+  }, [seedBankId, activeLocale, featureEnabled]);
+
+  const getTopBarButtons = () => {
+    const topBarButtons: TopBarButton[] = [];
+    if (selectedRows.length) {
+      topBarButtons.push({
+        buttonType: 'destructive',
+        buttonText: strings.DELETE,
+        // we don't want to delete locations that have active accessions
+        disabled: !selectedRows.some((location) => !location.activeAccessions),
+        onButtonClick: () => deleteStorageLocations(selectedRows.filter((location) => !location.activeAccessions)),
+      });
     }
-  }, [seedBankId, activeLocale]);
 
-  if (!storageLocations.length) {
+    return topBarButtons;
+  };
+
+  const onStorageLocationSelected = (location: PartialStorageLocation, fromColumn?: string) => {
+    if (fromColumn === 'name') {
```

Comment:
If you are only calling the 'onStorageLocationSelected' from the name column, this check wouldn't be necessary, but is ok to leave it